maintainers/maintainer-list.nix

```diff
@@ -16125,4 +16125,10 @@
     github = "ziguana";
     githubId = 45833444;
   };
+  rubyowo = {
+    name = "Rei Star";
+    email = "perhaps-you-know@what-is.ml";
+    github = "rubyowo";
+    githubId = 105302757;
+  };
```

Comment:
This is buried deep within the manual (and we should add it to CONTRIBUTING.md too so it's more discoverable for new maintainers...), but additions to `maintainers/maintainter-list.nix` are supposed to be in their own commits titled "maintainers: add [name]": https://nixos.org/manual/nixpkgs/unstable/#var-meta-maintainers

Reply:
ah okay thanks